.chloggen/clickhouseexporter-aggregation-temporality.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
it seems a breaking change. similar to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31803/files.

Reply:
Good catch, this would indeed break `INSERT`s for existing deployments

Reply:
Updated changelog entry `change_type` to `breaking`